frontend/app/src/index.tsx

```diff
@@ -28,6 +28,9 @@ import ThemedApp from "./ThemedApp"
 
 const engine = new Styletron({ prefix: "st-" })
 
+// Timestamp when React is loaded for GUEST_READY message
+window.ReactLoad = new Date()
```

Comment:
should we use `new Date().getTime()` for this and `guestReadyAt` for having a UTC timestamp that is easy to work with? I assume this date will be used in Snowsight so it probably does not matter too much, but I personally have always preferred to use timestamps instead Dates, so raising it here for sanity-checking 🙂

Reply:
Fair point - I defaulted to doing `new Date()` since that was the format specified in the dependency ticket and since Pat gave the PR a look

Reply:
Ah I didn't look closely enough at this before. Yes let's use `Date.now()` (equivalent to `new Date().getTime()`) to represent the date as UTC milliseconds since epoch. That is what we use in the SiS side.